src/04-modality-specific-files/03-electroencephalography.md

```diff
@@ -303,6 +333,14 @@ SHOULD be present:
 | material        | RECOMMENDED           | Material of the electrode  (for example, Tin, Ag/AgCl, Gold).          |
 | impedance       | RECOMMENDED           | Impedance of the electrode, units MUST be in `kOhm`.                   |
 
+{{ MACROS___make_columns_table(
+   {
+      "type": "RECOMMENDED",
```

Comment:
the electrode type is different from channel type --> for channel types, we have a list of restricted keywords, for electrode type, we don't (the latter is completely freeform text right now)

Reply:
I'll split it into `type_channels` and `type_electrodes` then. Thanks!